### PR TITLE
📚 Docs Refresh: Oracle Source

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Reorganization
- Airbyte Cloud/TLS note

### CODE REVIEW

File: docs/integrations/sources/oracle.md [hunk 1]
```diff
@@ -100,7 +61,7 @@ Your database user should now be ready for use with Airbyte.
 
 Case sensitive. Defaults to the upper-cased user if empty. If the user does not have access to the configured schemas, no tables will be discovered.
 
-## Connection via an SSH Tunnel
+## Connection via SSH Tunnel
```
Comment (marcosmarxm) [inline docs/integrations/sources/oracle.md:64]: Update feature table!

File: docs/integrations/sources/oracle.md [hunk 2]
```diff
@@ -117,6 +78,39 @@ Using this feature requires additional configuration, when creating the source.
 6. If you are using `Password Authentication`, then `SSH Login Username` should be set to the password of the User from the previous step. If you are using `SSH Key Authentication` leave this blank. Again, this is not the Oracle password, but the password for the OS-user that Airbyte is using to perform commands on the bastion.
 7. If you are using `SSH Key Authentication`, then `SSH Private Key` should be set to the RSA Private Key that you are using to create the SSH connection. This should be the full contents of the key file starting with `-----BEGIN RSA PRIVATE KEY-----` and ending with `-----END RSA PRIVATE KEY-----`.
 
+## Data type mapping
```
Comment (marcosmarxm) [inline docs/integrations/sources/oracle.md:81]: Data Type Mapping

**Review by marcosmarxm — APPROVED** [review]
LGTM small comments

File: docs/integrations/sources/oracle.md (hunk 1 above)
Comment (avaidyanatha) [reply docs/integrations/sources/oracle.md:64]: Done!

File: docs/integrations/sources/oracle.md (hunk 2 above)
Comment (avaidyanatha) [reply docs/integrations/sources/oracle.md]: Done.

**Pull request merged**